x-pack/plugins/triggers_actions_ui/public/application/context/alerts_context.tsx

```diff
@@ -16,7 +16,7 @@ export interface AlertsContextValue<MetaData = Record<string, any>> {
   http: HttpSetup;
   alertTypeRegistry: TypeRegistry<AlertTypeModel>;
   actionTypeRegistry: TypeRegistry<ActionTypeModel>;
-  toastNotifications: Pick<
+  toastNotifications?: Pick<
```

Comment:
We decided to keep toastNotifications as a required property, because it's the best option to show the server errors or messages to the user (post saving) and our upcoming PRs rely on this. 
I see some sort of additional work from your side of passing down this core toast, because we are expecting toastNotifications: core.notifications.toasts, but you use kibana services which has its own wrap function createNotifications over this. For the 7.8 we will try ti unify the usage of toasts, but for current moment, please pass it down from the core start.